docs/source/onboarding_resources/contributing/index.rst

```diff
@@ -43,25 +55,44 @@ The ``YOUR-BRANCH-NAME`` is the name of your branch, and you should try to
 be descriptive.  For example, if you're adding new documentation about an
 ischemic heart disease model, you might call your branch ``ihd-model``.
 
+Make sure you have pulled any recent updates from the main branch before 
+creating your new branch! 
+
 Modify the documentation
 ------------------------
 
 To make updates to the documentation, you should add or modify the
-``.rst`` files in the ``docs/source/`` directory.  Our documentation is written
+``.rst`` files in the ``vivarium_research/docs/source/`` directory.  Our documentation is written
 using `reStructuredText <http://docutils.sourceforge.net/docs/user/rst/quickref.html>`_
 and `sphinx <http://www.sphinx-doc.org/en/master/contents.html>`_.  Keep those
 links handy for reference.
 
-Once you've made your modifications, you can build them to see how they look.
+Use your favorite text editor for this part. If you want some recommendations on a text 
+editor from your teammates, check out the text editor section in this page on 
+:ref:`computing tools <computing_interfaces>`.
+
+Reviewing your changes
+----------------------
+
+Once you've made your modifications, you can build them to see how they look. 
+Often, people use Miniconda for building pages. More information can be found 
+in this page for :ref:`computing tools <computing_interfaces>`.
+
+Use these commands to build your page with edits: 
 
 ::
 
-   (vivarium_research) $> cd docs
+   (base) $> cd vivarium_research/docs
+   (base) $> conda activate vivarium_research ##This line is only needed if vivarium_research has not been activated yet
    (vivarium_research) $> make html
 
 This will create a new ``build`` sub-directory with the new documentation
-rendered in html.  You can open the file `docs/build/html/index.html` in your
-browser to view your changes.
+rendered in html.  You can open the file `vivarium_research/docs/build/html/index.html` in your
+browser to view your changes. 
+
+Note that some errors will appear as warnings in Miniconda, but will actually cause the build to 
```

Comment:
```suggestion
Note that any warnings in your terminal after the `make html` command will cause the build to 
```

This isn't a difference between local and GitHub per se, it's just that we have set up GitHub to fail on any warnings.